File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int len = nums.length;
        int ArraySum = 0;
        int ValueSum = 0;
        for(int i=1;i<=len;i++){
            ValueSum+=i;
        }
          for(int i=0;i<len;i++){
           ArraySum+=nums[i];
        }
        return ValueSum-ArraySum;

        
    }
}

File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int length = nums.size();
        int sum = 0;
        int natural_sum = 0;
        for(int i=0;i<length;i++){
            sum+=nums[i];
        }
        for(int i=1;i<=length;i++){
            natural_sum +=i;
        }
        return natural_sum-sum;
        
    }
};